src/stores/SqlQuery.tsx

```diff
@@ -0,0 +1,54 @@
+import {memoize} from 'lodash';
+import {ipaddress} from '../constants/constants';
+import getGlobalObjectStore from './GlobalObjectStore';
+
+class SqlQueryStore {
+
+  sqlDelete = (objid: number) => {
+    fetch(`${ipaddress}/api/delete`, {
+      method: 'POST',
+      headers: {'Content-Type': 'application/json'},
+      body: JSON.stringify({
+        id: objid,
+      }),
+    });
+    // .then(async res => await res.json());
+    // .catch(error => console.error('Error:', error))
+    // .then(async response => console.log('Success:', await response));
+  };
+
+  sqlGet = () => {
+    fetch(`${ipaddress}/api/get`, {
```

Comment:
as you can see for each call we are calling fetch, so let's make a helper that centralizes fetch.
so it will take the URL + the method + headers

Reply:
done